### PR TITLE
Bug 1986389: Limit the textarea not to exceed the parent window

### LINKED ISSUE #9598: the horizontal adjustment range of the text area exceeds the window
In the 'Create Project' window, the horizontal adjustment range of the 'Description' text area exceeds the window

openshift_bug

### PR DESCRIPTION
**Pull request by ut-wangbo**
In the 'Create Project' window, the horizontal adjustment range of the 'Description' text area exceeds the window.
Closes #9598

### CODE REVIEW

**Comment by spadgett** [comment]
/ok-to-test
/assign @rhamilto 

Thank you for the PR!

File: frontend/public/components/modals/_modals.scss [hunk 1]
```diff
@@ -40,6 +40,7 @@
   // so that input, textarea, button, and input-group-addon don't mask the inner scroll shadows
   input, textarea {
     &.pf-c-form-control {
+      max-width: 100%;
```
Comment (rhamilto) [inline frontend/public/components/modals/_modals.scss:43]: Nit.  We aim to alphabetize declarations by property in the console code base, so move this line after the background-color declaration below.

Comment (ut-wangbo) [reply frontend/public/components/modals/_modals.scss]: Okay. It has been moved as required.

**Pull request merged**